_components/validation/guidance/when-to-use.md

```diff
@@ -0,0 +1 @@
+**Provide helpful validation feedback.**  Give users immediate feedback on which validation requirements have been successfully met, and which have yet to be completed.
```

Comment:
Can we give an example? 

"For example, when a validation is not yet completed, send them a frowny face."

Reply:
I added a bit of clarity in https://github.com/uswds/uswds-site/pull/1851/commits/c796cd5243351f4580dfc0ab50ab2ea0a19190a1

Does that help?